tests/markov/hmm/test_integration.py

```diff
@@ -110,7 +110,7 @@ def test_hidden_transition_matrix(hmm_scenario):
     for perm in permutation_matrices(hmm_scenario.n_hidden):
         minerr = min(minerr, np.max(np.abs(perm.T @ model.transition_model.transition_matrix @ perm -
                                            hmm_scenario.msm.transition_matrix)))
-    np.testing.assert_almost_equal(minerr, 0, decimal=2)
+    np.testing.assert_almost_equal(minerr, 0, decimal=1)
```

Comment:
why can this test not hold up for decimals greater than 1?

Reply:
it does, occasionally, depending on where the maximum likelihood estimation of the HMM ends up. I relaxed the condition so the test does not spuriously fail

Reply:
might be worth adding a note for future devs